plugins/automation/spec/system/smoke_test_spec.rb

```diff
@@ -6,34 +6,6 @@
   fab!(:badge) { Fabricate(:badge, name: "badge") }
 
   before do
-    I18n.backend.store_translations(
-      :en,
-      {
-        discourse_automation: {
-          scriptables: {
-            test: {
-              title: "Test",
-              description: "Test",
-            },
-            something_about_us: {
-              title: "Something about us.",
-              description: "We rock!",
-            },
-            nothing_about_us: {
-              title: "Nothing about us.",
-              description: "We don't rock!",
-            },
-          },
-          triggerables: {
-            title: "Triggerable",
-            description: "Triggerable",
-            user_first_logged_in: {
-              description: "User first logged in.",
-            },
-          },
-        },
-      },
-    )
```

Comment:
We recently started to raise exceptions when we don't have a translation, I made various changes in this commit to avoid having to define these when writing specs.